src/components/NewMessage/NewMessage.vue

```diff
@@ -373,13 +386,34 @@ export default {
 		showAudioRecorder() {
 			return !this.hasText && this.canUploadFiles && !this.broadcast && !this.upload
 		},
+
 		showTypingStatus() {
 			return this.hasTypingIndicator && this.supportTypingStatus
 				&& this.settingsStore.typingStatusPrivacy === PRIVACY.PUBLIC
 		},
+
+		userAbsence() {
+			return this.chatExtrasStore.absence[this.token]
+		},
 	},
 
 	watch: {
+		conversation: {
+			immediate: true,
+			deep: true,
+			async handler(conversation) {
+				if (!conversation?.status || conversation.status !== 'dnd'
+					|| !this.isOneToOneConversation || this.chatExtrasStore.absence[this.token] !== undefined) {
```

Comment:
I'm checking here specifically for `undefined`. If user has standard DnD status, but without "Out of office" it will send a request to server each time you open the tab, and receive `null`. With that line, if stored value has `null`, request will be sent only once.

Once status message_id will be exposed (should be 'vacationing'), we can switch to checking it, and refactor this code
cc @nickvergessen should i raise an issue to track it? server or spreed repo?

Reply:
server, but it's unlikely to happen at this point